src/oc_bifrost/schema/sqitch.plan

```diff
@@ -1,6 +1,6 @@
 %syntax-version=1.0.0-b2
 %project=bifrost
-%uri=https://github.com/opscode/oc_bifrost
+%uri=https://github.com/chef/oc_bifrost
```

Comment:
This URI(https://github.com/chef/oc_bifrost) is looking like working with automate. Could you please update in all the sqitch.plan files. These changes can resolve https://github.com/chef/chef-server/issues/2803 issue